dashboard/shared/src/PersistentStateManager.ts

```diff
@@ -104,3 +108,9 @@ export class PersistentStateManager {
     }
   }
 }
+
+function boolFromString(s: string|null) {
+  if (s == "true") return true
```

Comment:
not sure that it is a safe transformation. user string maybe also `true`.

Reply:
True, but we can't know from the URL whether it's a string or bool. We can probably encode this somehow in a URL like `b"true"`, but I'm not sure it's worth it.